tests/test_attributes_managers.py

```diff
@@ -103,7 +103,7 @@ def perform_general_tests(attr_mgr, search_string):
     # attempt to remove templates that are locked
     removed_templates = attr_mgr.remove_templates_by_str(new_handle_stub, True)
     # verify that no templates were removed that have the new handle stub
-    assert 0 == len(removed_templates)
+    assert len(removed_templates) == 0
```

Comment:
A fan of auto-recoding I am not :)  Does it always reverse yoda conditions?

Reply:
It always recommend that constant literals are evaluated at the end of the boolean expressions for readability. o

I can disable it if needed, but the latter form is more readable. 

The fact that this one of only two YODA conditions in the entire repo shows that our own project style is pretty consistent.